docs/03-cli-reference/03-deploy.md

```diff
@@ -41,6 +41,12 @@ serverless deploy --stage production --region eu-central-1
 With this example we've defined that we want our service to be deployed to the `production` stage in the region
 `eu-central-1`.
 
+## List existing deploys
```

Comment:
Explanation text please, e.g. Something like "List recent deployments available in your S3 Deployment Bucket". Also mention to use the stage/region config in the provider config element.